svthalia/concrexit
# Prevent revoking bankaccounts if unbatched

Closes #940 

### Summary
Users now cannot revoke their mandate if there are still Thalia Pay payments unpatched or unprocessed. They are instructed to contact the board.

### How to test
1. Have a bank account added
2. Add a Thalia Pay payment
3. Check that revoking the mandate will display a warning message
4. Add payment to a batch
5. Check that revoking the mandate will still display a warning message
6. Process the batch
7. The mandate will be revoked.

Also see that the payment list now displays the processed-status of a user's payments.

Also contains a small bug fix on the admin widget (sorry I felt lazy to separate it but if you really want I can still do it)
